libpoolprotocols/stratum/EthStratumClient.h

```diff
@@ -57,6 +57,7 @@ class EthStratumClient : public PoolClient
 
 	void reset_work_timeout();
 	void processReponse(Json::Value& responseObject);
+	string processError(Json::Value& erroresponseObject);
```

Comment:
make it a ``std::string``

Reply:
Will do in next round (noted)
This PR is 0.14 to 0.15.
Original change was from Stratum Branch to 0.14